### ISSUE TITLE
review issues are visually dominated by instructions for reviewers

### ISSUE THREAD

**Issue by wojdyr** [openjournals/joss#305]
In particular getting as an author email telling me that it's important to seriously consider unsubscribing from GitHub notifications may be confusing (because I'm not subscribed). Note that this huge comment with two screenshots is not really relevant to the review issue. Perhaps it could be replaced with a link to a wiki page that would have detailed instructions?

**Comment by arfon** [comment]
Thanks for the feedback @wojdyr. We're always looking for ways to improve our processes. Some context below:

> Note that this huge comment with two screenshots is not really relevant to the review issue. Perhaps it could be replaced with a link to a wiki page that would have detailed instructions?

But it _is_ very relevant to the reviewer as they are automatically subscribed to these issues when they're created and the increase noise of GitHub notifications (from all future JOSS reviews) for these folks has been a significant pain point in the past.

We currently don't have a wiki set up for the project but I agree content like this could be posted there. I worry though if this were simply a link to a wiki page it would be more likely to be missed...

**Comment by Kevin-Mattheus-Moerman** [comment]
I agree with @wojdyr. Yesterday I tried to show off / demo the JOSS review system to some open source enthusiasts. It was confusing to have to scrolllllllllll down all the way. We could use an icon/picture/label e.g. at the top or bottom of the tick list to highlight a link to unsubscribing from the notifications?

**Comment by cMadan** [comment]
Maybe there's a way to make the link salient (e.g., a picture), but not necessarily take up so much screen space?

**Comment by danielskatz** [comment]
I'm going to close this as stale. If someone disagrees, they can certainly reopen it
